modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PythonClientCodegen.java

```diff
@@ -202,6 +209,16 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("git_push.sh.mustache", "", "git_push.sh"));
         supportingFiles.add(new SupportingFile("gitignore.mustache", "", ".gitignore"));
         supportingFiles.add(new SupportingFile("travis.mustache", "", ".travis.yml"));
+
+        if ("asyncio".equals(getLibrary())) {
```

Comment:
I was trying to replicate behavior I saw in the java client, where you can choose the library and the subdirectly seemingly populated from the resource directory with the same name.

If there's a better pattern here, I'd love to know.